### PR TITLE
Update mainnet and testnet urls in index page in service docs

### LINKED ISSUE #1627: Update http mainnet and testnet url in doc page
### Description
A documentation page in lisk-service is showing websocket endpoint in place of http endpoints

```
Current endpoints:
    main-net: wss://service.lisk.com/api/
    test-net: wss://testnet-service.lisk.com/api/

Expected endpoints:
    main-net: https://service.lisk.com/api/v2/.
    test-net:  https://testnet-service.lisk.com/api/v2/.
```

### Affected branch / URL
https://lisk.com/documentation/lisk-service/index.html#the-http-api

### Additional resources

### PR DESCRIPTION
**Pull request by chris529**
Closes #1627

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -108,7 +109,7 @@ Lisk Testnet::
 .Example request: Obtain a list of all 101 currently forging delegates
```
Comment (sameersubudhi) [inline modules/ROOT/pages/index.adoc]:
```suggestion
.Example request: Obtain a list of generators in the current round
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -108,7 +109,7 @@ Lisk Testnet::
...
 [source,bash]
 ----
-curl -X GET "http://localhost:9901/api/v2/forgers" -H  "accept: application/json"
+curl -X GET "http://localhost:9901/api/v3/forgers" -H  "accept: application/json"
```
Comment (sameersubudhi) [inline modules/ROOT/pages/index.adoc]:
```suggestion
curl -X GET "http://localhost:9901/api/v3/generators" -H  "accept: application/json"
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
The API has been updated. Request you to kindly make the following changes to reflect the same.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by TalhaMaliktz — COMMENTED** [review]
The changes LGTM, however, I was not able to access the dynamic pages/content generated by swagger. Can you access them?

**Comment by sameersubudhi** [comment]
@TalhaMaliktz The mainnet migration hasn't happened yet. So, deployments are not yet live. Hence, you won't be able to access them! 🙂

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

**Comment by Tschakki** [comment]
@chris529 I think there was a misunderstanding, the issue #1627 is actually referring to a bug in our currently live documentation. So please update the links according to the 0.7 version, thanks!

**Pull request merged**